server/db/User.js

```diff
@@ -156,9 +149,9 @@ User.verifyByTokenIfAdmin = async function (token) {
 };
 
 User.afterCreate(async (user) => {
-  const cart = await Cart.create();
+  const [cart] = await Cart.findOrCreate({ where: { userId: user.id } });
   await user.setCart(cart);
-  console.log(cart.userId === user.id);
+  // console.log(cart.userId === user.id);
```

Comment:
Had to update this for the new `POST /api/users` route, because we don't always want to create a new cart when creating a User